guides/coco-dev.md

```diff
@@ -0,0 +1,245 @@
+# Running a workload
+
+## Creating a sample CoCo workload
+
+Once you've used the operator to install Confidential Containers, you can run a pod with CoCo by simply adding a runtime class.
+First, we will use the `kata-qemu-coco-dev` runtime class which uses CoCo without hardware support.
```

Comment:
Are we trying to support 0.10.0 here? If so we probably should make additional changes to reflect the `kata-qemu-coco-dev` runtime class being used for non-TEEs, if not then we shouldn't mention it here?

Reply:
I believe we are, I've made the appropriate changes